packages/log4j2-hotpatch/0001-Log-to-stderr-instead-of-stdout.patch

```diff
@@ -0,0 +1,35 @@
+From 61921bad328e3b12305fd91162c30e9a54bedfbf Mon Sep 17 00:00:00 2001
+From: Stewart Smith <trawets@amazon.com>
+Date: Thu, 16 Dec 2021 07:54:47 -0800
+Subject: [PATCH] Log to stderr instead of stdout
```

Comment:
Should we also suppress the log output entirely?  Otherwise it will be visible in container logs, and if those are being parsed it could be problematic.

(I think we can suppress the log output entirely with another JVM option to the hotpatch in hotdog.)

Reply:
Dropping this patch in favor of suppressing output in hotdog.

Reply:
Corresponding hotdog PR: https://github.com/bottlerocket-os/hotdog/pull/1